schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -832,7 +832,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2015-11-01-preview/Microsoft.OperationsManagement.json#/resourceDefinitions/ManagementConfigurations" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-11-01-preview/Microsoft.OperationsManagement.json#/resourceDefinitions/solutions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Peering.json#/resourceDefinitions/peerings" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Peering.json#/resourceDefinitions/peerings_prefixes" },
```

Comment:
>peerings_prefixes [](start = 128, length = 17)

this shouldn't be removed. I can see the resource type in Microsoft.Peering.json.